lld/test/ELF/x86-64-dwarf5-64-debug-names-type-comdat.test

```diff
@@ -0,0 +1,97 @@
+# REQUIRES: x86
+
+# RUN: llvm-mc -filetype=obj -triple=x86_64-unknown-linux --defsym DWARF32=1 %s -o %t.o
+# RUN: llvm-mc -filetype=obj -triple=x86_64-unknown-linux --defsym DWARF32=1 %s -o %t1.o
```

Comment:
You can do `ld.lld %t.o %t.o`.

`-triple=x86_64` since there is no Linux specific stuff. We omit the OS to essentially say the test is generic to all ELF OSes.